flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/table/IndexGeneratorTest.java

```diff
@@ -86,6 +89,7 @@
                                     LocalDateTime.of(2020, 3, 18, 12, 12, 14, 1000)),
                             (int) LocalDate.of(2020, 3, 18).toEpochDay(),
                             (int) (LocalTime.of(12, 13, 14, 2000).toNanoOfDay() / 1_000_000L),
+                            TimestampData.fromTimestamp(Timestamp.valueOf("2020-03-18 12:12:14")),
```

Comment:
You would need to use a custom timezone to test correctly the TIMESTAMP_WITH_LOCAL_TIME_ZONE:
```
TimestampData.fromInstant(
                LocalDateTime.of(years, months, days, hours, minutes, seconds, nanos)
                        .atZone(<MyCustomTestZone>) // i.e. ZoneId.of("Asia/Shanghai")
                        .toInstant());
```